tests/run-make/core-no-fp-fmt-parse/rmake.rs

```diff
@@ -0,0 +1,20 @@
+// This test checks that the core library of Rust can be compiled without enabling
+// support for formatting and parsing floating-point numbers.
+
+extern crate run_make_support;
+
+use run_make_support::rustc;
+use std::path::PathBuf;
+
+fn main() {
+    rustc()
+        .arg("--edition")
+        .arg("2021")
+        .arg("-Dwarnings")
+        .arg("--crate-type")
+        .arg("rlib")
+        .arg_path(&["..", "..", "..", "library", "core", "src", "lib.rs"])
+        .arg("--cfg")
```

Comment:
We have a cfg helper already I think that you could use here

Reply:
You're right, this is cleaner!

```rust
.cfg("no_fp_fmt_parse")
```